README.md

```diff
@@ -200,13 +200,18 @@ server->on("/settings.html", HTTPMethod::HTTP_GET, [server](){
 
 # Endpoints
 
-### GET /
+
+## GET /
+
++ AP and API mode
```

Comment:
Aesthetic only: what about `###### *(AP,API)*` instead? Might be a little cleaner.

Reply:
pushed